profiles/models.py

```diff
@@ -297,9 +311,19 @@ class VerifiedPersonalInformationPermanentForeignAddress(
 ):
     RELATED_NAME = "permanent_foreign_address"
 
-    street_address = fields.EncryptedCharField(max_length=1024, blank=True)
-    additional_address = fields.EncryptedCharField(max_length=1024, blank=True)
-    country_code = fields.EncryptedCharField(max_length=3, blank=True)
+    street_address = fields.EncryptedCharField(
+        max_length=1024,
+        blank=True,
+        help_text="Street address or whatever is the _first part_ of the address.",
```

Comment:
I think admin doesn't support "\_xxx\_" style emphasis. Consider removing the underscores or replacing with `<em>xxx</em>`

Reply:
That could be true. I have only checked how these texts look in the automatically generated GraphQL documentation and that seems to support markdown.